UnitsNet/CustomCode/UnitParser.cs

```diff
@@ -29,15 +29,15 @@
 
 namespace UnitsNet
 {
-    internal delegate TUnit ParseUnit<out TUnit>(string value, string unit, IFormatProvider formatProvider = null);
+    internal delegate TQuantity ParseUnit<out TQuantity>(string value, string unit, IFormatProvider formatProvider = null);
 
     internal static class UnitParser
     {
         [SuppressMessage("ReSharper", "UseStringInterpolation")]
-        internal static TUnit ParseUnit<TUnit>([NotNull] string str,
+        internal static TQuantity ParseUnit<TQuantityEnum, TQuantity>([NotNull] string str,
```

Comment:
I renamed the type parameters to match a [naming convention](https://github.com/anjdreas/UnitsNet/issues/260).
Instead of `TQuantity, TUnit` it is now `TQuantityEnum, TQuantity`.